cfdp-core/src/transaction/recv.rs

```diff
@@ -99,11 +118,10 @@ impl<T: FileStore> RecvTransaction<T> {
             status: TransactionStatus::Undefined,
             config,
             filestore,
-            transport_tx,
             message_tx,
             file_handle: None,
             saved_segments: Vec::new(),
-            naks: VecDeque::new(),
+            nak_procedure: NakProcedure::Deferred,
```

Comment:
Now that nak_procedure is a field of `EntityConfig` should this field be passed in the constructor from the entity config?

Reply:
Yes, sorry, I didn't know how I should configure that, please change it as you think it should be done.

Reply:
Okay no problem. I can make a tweak PR after this gets merged.